opentelemetry/proto/common/v1/common.proto

```diff
@@ -53,3 +53,10 @@ message StringKeyValue {
   string key = 1;
   string value = 2;
 }
+
+// LibraryInfo is a message representing the instrumented library informations such as the fully
```

Comment:
Should this be
```suggestion
// LibraryInfo is a message representing the instrumenting library informations such as the fully
```
and represent the information that comes from the "named tracer"?

Otherwise, where would this information come from?

Reply:
It can be both instrumenting and instrumented. Think if a library has a dependency on otel and instruments itself (not via a plugin).

Reply:
A library *can* be both. But that is a special case and in all cases where it's different using the "instrument**ed**" library is wrong.